app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.java

```diff
@@ -204,7 +237,10 @@ public void onImageClicked() {
 
             @Override
             public void onCallToActionClicked() {
-                if (callToActionIsTranslation ? (callToActionTargetSummary != null && callToActionSourceSummary != null) : callToActionSourceSummary != null) {
+                if (imageEditType == ImageEditType.ADD_TAGS) {
+                    getActivity().startActivityForResult(SuggestedEditsImageTagEditActivity.Companion.newIntent(getActivity(), imagePage), ACTIVITY_REQUEST_IMAGE_TAGS_EDIT);
+                }
```

Comment:
Should we use `} elseif ` here?

Reply:
good point @cooltey . Will use return; instead like our regular pattern.